### DISCUSSION TITLE
Why doesn't useFetch have an initialCache option?

### DISCUSSION THREAD

**Discussion by Vivi-Ornitier** [nuxt/nuxt#17363] [Questions] [answered]
`useAsyncData` has an `initialCache` option.

I want to use `useFetch` most of the time,
but I have to use `useAsyncData` because there is no `initialCache` option.

Why doesn't `useFetch` have an `initialCache` option?

**Comment by Flanker72** [comment, marked as answer]
It's not covered by docs, but `useFetch` HAS the `initialCache` option. `useFetch` is a wrapper around the `useAsyncData` and `$fetch` - so, it has this feature (https://github.com/nuxt/framework/blob/b1470c081949b3287a8d66f0934818192d97bec9/packages/nuxt/src/app/composables/fetch.ts#L14).

**Reply by Vivi-Ornitier** [reply]
Thank you for your reply.
It's worked!
